packages/config/config/devices/0x0063/35931.json

```diff
@@ -30,6 +30,10 @@
 				{
 					"label": "Last brightness setting",
 					"value": 1
+				},
+				{
+					"label": "Last lamp status",
+					"value": 2
 				}
```

Comment:
So 1 is the last non-zero brightness and 2 is the previous state?

Reply:
Correct. With `2`, a lamp that is off will remain off. If the lamp is on before power is cycled, the behavior is the same as `1`.